src/NovelRT/Audio/AudioService.cpp

```diff
@@ -7,8 +7,9 @@ AudioService::AudioService() :
   _device(Utilities::Lazy<std::unique_ptr<ALCdevice, void(*)(ALCdevice*)>> (std::function<ALCdevice*()>([this] {
     auto device = alcOpenDevice((_deviceName.empty())? nullptr : _deviceName.c_str());
     if (!device) {
-      _logger.logError("OpenAL device creation failed! {}", getALError());
-      throw std::runtime_error("OpenAL failed to create an audio device! Aborting...");
+      std::string error = getALError();
+      _logger.logError("OpenAL device creation failed! {}", error);
+      throw Exceptions::InitialisationFailureException("OpenAL failed to create an audio device! Aborting...", error);
```

Comment:
InitialisationFailureException should have a base, uniform message. The error should then be the only arg (we could prepend `error` with custom text that way and it would make for better API UX).